### DISCUSSION TITLE
Top languages card is empty

### DISCUSSION THREAD

**Discussion by 2huiju** [anuraghazra/github-readme-stats#1937] [Q&A] [answered]
Describe the bug I have created my Vercel instance and am using it but my languages card does not show anything.
Expected behaviour The languages card displays something.
Screenshots / Live demo link (paste the github-readme-stats link as markdown image) 

It used to work properly, but it hasn't worked since a few days ago
The terms of use also included a private repository, but it does not work.

![Most Used Languages](https://user-images.githubusercontent.com/87798108/183889867-8340db64-5440-4304-91f5-3dadd81ab453.png)

**Comment by rickstaa** [comment, marked as answer]
@2huiju Thanks for your issue. I checked your account, and it looks like the results shown on the card are correct. You can see this in action by using the following query in the [graphql explorer](https://docs.github.com/en/graphql/overview/explorer):

```graphql
query userInfo($login: String!) {
  user(login: $login) {
    # fetch only owner repos & not forks
    repositories(ownerAffiliations: OWNER, isFork: false, first: 100) {
      nodes {
        name
        languages(first: 10, orderBy: {field: SIZE, direction: DESC}) {
          edges {
            size
            node {
              color
              name
            }
          }
        }
      }
    }
  }
}
```

You can then use the following in the query variables field.

```graphql
{"login": "2huiju"}
```

There is only one repository that shows up, namely `2huiju`. This repository, however, do not have any language info attached to them. This is probably because they only contain languages that are not regarded by [linguist ](https://github.com/github/linguist)as languages (see https://github.com/github/linguist/issues/5951 for more info). Further, please be aware that we currently do not fetch data from forks only from parent repositories and that the public Vercel instance does not have access to your private repositories (see [the FAQ](https://github.com/anuraghazra/github-readme-stats/discussions/1770) for more info).

**Reply by rickstaa** [reply]
@2huiju Did my answer solve your problem?
